app/css/popup.css

```diff
@@ -46,9 +88,327 @@ body {
 /**
  * Override Bootstrap responsive because Tab Wrangler is not a mobile UI.
  */
-
 @media (min-width: 576px) {
   .form-inline .form-check {
-    justify-content: flex-start;
+    -webkit-box-pack: start;
+        -ms-flex-pack: start;
+            justify-content: flex-start;
   }
 }
+
+/**
+ * Extension buttons
+ */
+.tw-button {
```

Comment:
Do the `.tw-` classes come from another framework? Since this is an open source project I want to make sure all other open source projects are attributed properly. Anything installed with NPM will have its appropriate license in the `node_modules/` directory.

Does this framework work well with Bootstrap too? I'm concerned some of the Bootstrap classes won't work as intended when we mix with another full framework.

Reply:
`.tw-` is TabWrangler's styles